controllers/OrganisationsController.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('./BaseController');
+const { orgLookup } = require('../services/OrganisationSearchService');
+
+class OrganisationsController extends BaseController {
```

Comment:
Looking at the other controllers, it looks like it's a one-to-one relationship with routes. If we need to have another controller for when we come to list all organisations (the LEV admin view), I think this needs to be the singular form "organisation".

Reply:
Done